### PR TITLE
Fix lint error in register.go

### LINKED ISSUE #102: Fix lint error in register.go
File: [register.go](https://github.com/openebs/node-disk-manager/blob/master/pkg/apis/openebs.io/register.go)

```
Line 4: warning: exported const GroupName should have comment (or a comment on this block) or be unexported (golint)
```

### PR DESCRIPTION
**Pull request by andreathniah**
fixes #102 
fix lint error by adding the comment on exported const group name
Signed-off-by: Andrea Thniah <andreathniah@gmail.com>

### CODE REVIEW

File: pkg/apis/openebs.io/register.go [hunk 1]
```diff
@@ -1,5 +1,6 @@
 package openebsio
 
 const (
+	// Point GroupName to the openebs site
```
Comment (sagarkrsd) [inline pkg/apis/openebs.io/register.go]: Thanks @andreathniah for this PR !
Can you please change it to something like "Use openebs.io as a GroupName" as openebs.io here is not a site.

Comment (andreathniah) [reply pkg/apis/openebs.io/register.go]: Woosp @sagarkrsd! I pushed another commit that changed it!

**Comment by sagarkrsd** [comment]
@andreathniah Can you please fix the DCO signoff error !

**Comment by sagarkrsd** [comment]
@andreathniah I think you have to rebase all the commits and send it into one commit with DCO sign-off.

**Comment by andreathniah** [comment]
> e all the commits and send it into one commit with DCO

I rebased and add the sign-off message for each of the commits. I'm not sure why it still doesn't seem to work?

**Comment by sagarkrsd** [comment]
Thanks @andreathniah . Can you please do "git commit -s --ammend " ?

**Comment by andreathniah** [comment]
> Thanks @andreathniah . Can you please do "git commit -s --amend " ?

That worked! Thank you for the help, apologies for the countless mistakes made!

**Comment by sagarkrsd** [comment]
Great @andreathniah . Good job.
Thanks for your PR !

File: pkg/apis/openebs.io/register.go [hunk 2]
```diff
@@ -1,5 +1,6 @@
 package openebsio
 
 const (
+	// Use openebs.io as a GroupName
```
Comment (a4abhishek) [inline pkg/apis/openebs.io/register.go]: Please make this comment in such form so that it begins with `GroupName...`

**Comment by sagarkrsd** [comment]
Thanks @a4abhishek .
@andreathniah Just one last change as @a4abhishek suggested, Please make it something like "GroupName is the constant used for ...."

**Pull request merged**